README.md

```diff
@@ -245,8 +245,7 @@ echo '
 {
     "email": "TEST@EXAMPLE.COM" ,
     "username" : "USER_NAME",
-    "phone_number" : "12345",
-    "roles": ["ROLE_1", "ROLE_2"]
+    "phone_number" : "12345"
```

Comment:
sorry but then what you're doing is not a PUT anymore,  as PUT is suppose to REPLACE completely the resource by an other.  then what you want to do is an additional PATCH method  where only precised fields get updated.